ambry-server/src/test/java/com/github/ambry/server/StatsManagerTest.java

```diff
@@ -654,19 +654,17 @@ public void testReplicaFromOfflineToDropped() throws Exception {
     Map<String, StatsSnapshot> accountContainerPairMap = new HashMap<>();
     long totalSize = 0;
     for (int i = 0; i < random.nextInt(MAX_ACCOUNT_COUNT - MIN_ACCOUNT_COUNT + 1) + MIN_ACCOUNT_COUNT; i++) {
-      String accountIdStr = "A[".concat(String.valueOf(i)).concat("]");
       Map<String, StatsSnapshot> containerMap = new HashMap<>();
       long subTotalSize = 0;
       for (int j = 0; j < random.nextInt(MAX_CONTAINER_COUNT - MIN_CONTAINER_COUNT + 1) + MIN_CONTAINER_COUNT; j++) {
-        String containerIdStr = "C[".concat(String.valueOf(j)).concat("]");
         long validSize = random.nextInt(2501) + 500;
         subTotalSize += validSize;
-        containerMap.put(containerIdStr, new StatsSnapshot(validSize, null));
-        accountContainerPairMap.put(accountIdStr + Utils.ACCOUNT_CONTAINER_SEPARATOR + containerIdStr,
+        containerMap.put(Utils.statsContainerKey((short) j), new StatsSnapshot(validSize, null));
```

Comment:
Thanks for making this change. One minor suggestion: I see there are lots of places where `int`  is casted to `short`. Can we add same method something like this:
```
  public static String statsContainerKey(int containerId) {
    return statsContainerKey((short) containerId);
  }
```

Reply:
I think it's better not to do this, container id and account id should stay short. This casting is because in test cases, we use int to iterate from 0 to some number.